src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -220,7 +220,10 @@ private void updateExistingMilestone(Optional<PickerMilestone> existingMilestone
         HBox assignedMilestoneBox = createPreviouslyAssignedMilestoneBox();
         assignedMilestonePane.getChildren().add(assignedMilestoneBox);
 
-        if (!existingMilestone.isPresent()) return;
+        if (!existingMilestone.isPresent()){
+            assignedMilestoneBox.setStyle("-fx-border-radius: 3;-fx-border-style: dotted;-fx-alignment:center");
```

Comment:
I think @damithc intended for the dotted lines to appear if any of the labels' spaces are empty. i.e. this should also be in `updateNewlyAssignedMilestone` to handle the case when `selectedMilestone` is not present.